acceptance/config/aio/options.rb

```diff
@@ -1,6 +1,7 @@
 {
   :type => 'aio',
   :is_puppetserver => true,
+  :puppetserver_confdir => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
Also I just remembered that beaker uses `puppetserver_confdir` when bouncing the puppetserver: https://github.com/puppetlabs/beaker/blob/beaker2.5.0/lib/beaker/dsl/helpers.rb#L551. Will moving this to the options file instead of the host files cause beaker to fall back to its hardcoded (and incorrect) AIO defaults?